server/dto/node_stats_request.go

```diff
@@ -1,6 +1,6 @@
 package dto
 
 type NodeStatsRequest struct {
-	NodeKey  string         `json:"node_key"`
-	Sessions []SessionStats `json:"sessions"`
```

Comment:
You can remove `NodeStatsRequest.Sessions` fields at all

Reply:
It's currently being used and is required by api. It could be removed from everywhere, since it's empty array at the moment, but it will require changing api as well, and might be worth to do this in https://mysteriumnetwork.atlassian.net/browse/MYST-92